src/CloudFoundry.VisualStudio/ProjectPush/PublishProfileEditorResources.cs

```diff
@@ -668,5 +669,32 @@ protected void RaisePropertyChangedEvent(string propertyName)
                 handler(this, new PropertyChangedEventArgs(propertyName));
             }
         }
+
+        public bool ValidateRoute(string hostName)
```

Comment:
Should be ValidateRoutes() and it should validate the routes within the datacontext.